crypto/etls/listener.go

```diff
@@ -22,41 +22,41 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type CipherHandler func(conn net.Conn) (cryptoConn *CryptoConn, err error)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CipherHandler should have comment or be unexported